py2many/cli.py

```diff
@@ -386,26 +451,10 @@ def _process_dir(settings, source, outdir, env=None, _suppress_exceptions=True):
         target_path = outdir / relative_path
         target_dir = target_path.parent
         os.makedirs(target_dir, exist_ok=True)
+        input_paths.append(path)
 
-        try:
-            if _process_once(settings, path, target_dir, env=env):
-                successful.append(path)
-            else:
-                print(f"Error: Could not reformat: {path}")
-                format_errors.append(path)
-        except Exception as e:
-            print(f"Error: Could not transpile: {path}")
-            if isinstance(e, AstNotImplementedError):
-                print(f"Due to {e.lineno}:{e.col_offset}: {e.__class__.__name__} {e}")
-            else:
-                print(f"Due to: {e.__class__.__name__} {e}")
-            failures.append(path)
-            if _suppress_exceptions:
-                if _suppress_exceptions is not True:
-                    if not isinstance(e, _suppress_exceptions):
-                        raise e
-            else:
-                raise e
+    successful = _process_many(settings, input_paths, outdir, env=env)
+    failures = set(input_paths) - set(successful)
```

Comment:
this is "wrong".

failures has previously meant failed to transpile.  Not failed to format.

`_process_many` needs to return `format_errors` and `successful`

Reply:
https://github.com/adsharma/py2many/pull/335 has extra assertions which verify that `successful` is being populated correctly.

Reply:
Yes - I thought about it, but felt we could defer it until we figure out how to invoke the formatters for the _process_many() case.

Ideally, I'd like slow formatters like ktlint and format.jl to be invoked once on N files for good user experience. But I'm not sure if we have enough error parsing logic to figure out which files had format errors.